api/exec/update_env.go

```
@@ -13,15 +18,36 @@ func UpdateEnv(workspaceId string, applicationId string, key string, value strin
+}
+
+func UpdateEnvWithLabel(workspaceId string, labels []string, key string, value string) error {
+	header := make(map[string]string)
+	accessToken, err := GetAccessToken()
+	if err != nil {
+		return err
+	}
+	header["Authorization"] = "Bearer " + accessToken
+
+	updateEnvReq := updateEnvRequest{NewValue: value}
+	requestJson, err := json.Marshal(updateEnvReq)
```

golangci-lint

ineffectual assignment to err

(ineffassign)